src/pages/__tests__/cats-get.test.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env jest */
+import React from 'react'
+import { shallow } from 'enzyme'
+import redirect, { directToAppExtension } from 'src/utils/redirect'
+import localStorageMgr from 'src/utils/local-storage'
+import { STORAGE_REFERRAL_DATA_REFERRING_CHANNEL } from 'src/utils/constants'
+
+jest.mock('src/utils/redirect')
+jest.mock('src/utils/local-storage')
+
+const getMockProps = () => ({
+  location: {
+    search: '',
+  },
+})
+
+afterEach(() => {
+  jest.clearAllMocks()
+})
+
+describe('GetExtensionRedirectPage', () => {
+  it('renders without error', () => {
+    const GetExtensionRedirectPage = require('../cats/get').default
+    const mockProps = getMockProps()
+    shallow(<GetExtensionRedirectPage {...mockProps} />)
+  })
+
+  it('redirects to the proper browser extension', () => {
+    const GetExtensionRedirectPage = require('../cats/get').default
+    const mockProps = getMockProps()
+    shallow(<GetExtensionRedirectPage {...mockProps} />)
+    expect(redirect).toHaveBeenCalled()
+    expect(directToAppExtension).toHaveBeenCalled()
```

Comment:
A better test: mock a return URL for `directToAppExtension`, then assert redirect is called with that URL.